src/tools/web.ts

```diff
@@ -0,0 +1,138 @@
+import { z } from "zod";
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import { errorToCallToolResult } from "../lib/mcp";
+import { SrchdError } from "../lib/error";
+import Firecrawl from "@mendable/firecrawl";
+import { WEB_SERVER_NAME as SERVER_NAME } from "./constants";
+
+const SERVER_VERSION = "0.1.0";
+
+export async function createWebServer(): Promise<McpServer> {
+  const server = new McpServer({
+    name: SERVER_NAME,
+    title: "Web Browsing & Search",
+    description: "Tools to search and browse the web",
+    version: SERVER_VERSION,
+  });
+
+  // The Firecrawl SDK dosen't fetch the key automatically from env.
+  const firecrawl = new Firecrawl({ apiKey: process.env.FIRECRAWL_API_KEY });
+
+  server.tool(
+    "fetch",
+    "Returns a Markdown-formatted content of the webpage at url.",
+    {
+      url: z.string().describe("The URL of the webpage to fetch."),
+      offset: z
+        .number()
+        .describe(
+          "The offset (in number of characters) of the content to fetch. (Default: 0)",
```

Comment:
Inconsistent with the way you render default max below should be 

(default: 0).